app/code/Magento/Wishlist/Api/data/WishlistInterface.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Magento\Wishlist\Api\data;
```

Comment:
First letter in namespace should be uppercase - `Magento\Wishlist\Api\Data`

Reply:
Resolved